### ISSUE TITLE
Is it possible to create REST API with this package? HELP

### ISSUE THREAD

**Issue by ajitdas123** [jeremykenedy/laravel-auth#94]
Hello, 
i was just wondering is it possible to create REST API with authentication system with this package ?

I have tried with this
`Route::group(['middleware'=>'auth:api'], function(){
    Route::get('hello','ApiTestControler@index');
});
`
and tried to access this with this url in postman `http://localhost:8000/api/hello?token=16Wf313JyO66cgXk6UstxsIzrf0hlt8tsP09i04UczkWysHWoveVcHGhUPz6YTdX`

and its showing me login page
So how can i authenticate an user using api token and then do the task 

Basically i want user to be able to login into mobile app using social account such as Facebook/google 
And then using their id or something i want to authenticated them on my server and send them `access token` which will  allow them to access the rest of the application
Any help is appreciated 
Thanks

**Comment by jeremykenedy** [comment]
The token that it generates is for account confirmation and is not an api token. If you want to use an oath2 system to make api calls you will need to use a jwt package or passport to setup that kind of api with system. 

I would say installing laravel passport is going to be the easiest and fastest way to achieve your goal.
